posts/2020-05-15-five-years-of-rust.md

```diff
@@ -0,0 +1,508 @@
+---
+layout: post
+title: "Five Years of Rust"
+author: The Rust Core Team
+---
+
+With all that's going on in the world you'd be forgiven for forgetting that as
+of today, it has been five years since we released 1.0 in 2015! Rust has changed
+a lot these past five years, so we wanted reflect back on all of our
+contributors' work since the stabilization of the language.
+
+**If this is the first time you're reading about Rust:** Rust is a general
+purpose programming language empowering everyone to build reliable and efficient
+software. Rust can be built to run anywhere in the stack, whether as the kernel
+for your operating system or your next web app. It is built entirely by an open
+and diverse community of individuals, primarily volunteers who generously donate
+their time and expertise to help make Rust what it is.
+
+## Major Changes since 1.0
+
+#### 2015
+
+**[1.2] — Parallel Codegen** Compile time improvements are large theme to every
+release of Rust, and it's hard to imagine that there was a short time where
+Rust had no parallel code generation at all.
+
+**[1.3] — The Rustnomicon** Our first release of the fantastic "Rustnomicon", a
+book that explores Unsafe Rust and its surrounding topics has become a great
+resource for anyone looking to learn and understand one of the hardest aspects
+of the language.
+
+**[1.4] — Windows MSVC Tier 1 Support** The first tier 1 platform promotion was
+bringing native support for 64-bit Windows using the Microsoft Visual C++ toolchain
+(MSVC). Before 1.4 you needed to also have MinGW (a third party GNU environment)
+installed in order to use and compile your Rust programs. Rust's Windows support
+is one of the biggest improvements these past five years. Just recently
+Microsoft [announced a public preview of their official Rust support for the
+WinRT API!][winrt] Now it's easier than ever build top quality native and cross
+platform apps.
+
+[winrt]: https://blogs.windows.com/windowsdeveloper/2020/04/30/rust-winrt-public-preview/
+
+**[1.5] — Cargo Install** The addition of being able to build Rust binaries
+alongside cargo's pre-existing plugin support has given birth to an entire
+ecosystem of apps, utilities, and developer tools that the community has come
+to love and depend on. Quite a few of the commands cargo has today were first
+plugins that the community built and shared on crates.io!
+
+#### 2016
+
+**[1.6] — Libcore** Libcore is a subset of the standard library that only
```

Comment:
I find the capitalization odd here. At least in the text, maybe `libcore` in code font would make more sense (similar to `cdylib` below)?